mem/ctrl/CtrlMemDynamicRTL.py

```diff
@@ -33,9 +33,9 @@ def construct(s, CtrlPktType, CtrlSignalType, ctrl_mem_size,
     # assert( ctrl_mem_size <= total_ctrl_steps )
 
     # Constant
-    CtrlAddrType = mk_bits(clog2(max(ctrl_mem_size, ctrl_count_per_iter)))
-    PCType = mk_bits(clog2(ctrl_count_per_iter + 1))
-    TimeType = mk_bits(clog2(total_ctrl_steps + 1))
+    CtrlAddrType = mk_bits(clog2(ctrl_mem_size))
+    PCType = mk_bits( clog2(ctrl_mem_size + 1) )
+    TimeType = mk_bits(8)
```

Comment:
Add `MAX_CTRL_COUNT = 1024` after [here](https://github.com/tancheng/VectorCGRA/blob/32d19cef451342f8c02af1d84a945245c70ffa10/lib/util/common.py#L27) (i.e., 10-bit), change this to `TimeType = mk_bits(clog2(MAX_CTRL_COUNT))`, and import that file in this `.py`.